spec/requests/api/internal/v2/hearing_event_logs_request_spec.rb

```diff
@@ -63,22 +63,28 @@
         end
       end
 
-      context "when no hearing log is found" do
+      context "when no Hearing ID found" do
```

Comment:
```suggestion
      context "when no data is found for the given Hearing ID" do
```

Reply:
This change is intentional: when the ID of the hearing is not found, it returns 500 errors.
In case no data is found for the hearing, it returns 200. 
We already have another test with 200: https://github.com/ministryofjustice/laa-court-data-adaptor/blob/2399a564f4a6d5ad5c6aa893c937603cc2878f53/spec/requests/api/internal/v2/hearing_event_logs_request_spec.rb#L35

Maybe I could add a new one with 200 and empty returns, but it does not add anything to the test on #L35

Reply:
Ah ok, thanks!